src/pages/background/handlers/browser-event.ts

```diff
@@ -7,8 +7,10 @@ export const onBeforeSendHeadersHandler = async (
   console.log("details", details);
   let headers = await getSignifyHeaders(
     details.originUrl!,
+    details.url,
```

Comment:
**onBeforeSendHeadersHandler** call is specific to firefox, this call intercepts the network request on firefox and does not work on chrome. We might need to change this behaviour as now we expect additional information such as reqInit and requested url.